contracts/contracts/ValidatorProxy.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.5.8;
+
+
+contract ValidatorProxy {
```

Comment:
I think the proxy should follow this interface, right? https://github.com/poanetwork/poa-bridge-contracts/blob/master/contracts/IBridgeValidators.sol